### ISSUE TITLE
not working in ie11 for my custom font

### ISSUE THREAD

**Issue by drewB** [typekit/webfontloader#219]
I am running into an issue where my custom font is properly detected in chrome, firefox, safari but not ie11.  Here is my code (in coffeescript):

    WebFont.load
      custom: 
        families: ['Flaticons Solid']
        testStrings: 'Flaticons Solid': "\ue0ca"
      fontactive: (familyName, fvd) =>
        @myMethod()
      fontinactive: (familyName, fvd) ->
        console.log "can't load font #{familyName}"

In ie11, it doesn't load.

**Comment by drewB** [comment]
I have routed around a bunch and it looks like the test string is the same width in my custom font and the fallbacks even when it is properly loaded in ie11.

**Comment by drewB** [comment]
I should also note is actually works in ie8.

**Comment by drewB** [comment]
Yes, it appears to be specific to the unicodes in the custom font.  Using a different font that uses a different unicode range works as expected.

**Comment by drewB** [comment]
actually looks like I was mistake about it being the unicodes.  I was able to get it to work with FontAwesome which uses fXXX for its unicodes so thought that was the issue.  However, I tried building a new custom font in that range and it is still not working.

**Comment by bramstein** [comment]
Thanks for reporting this. It sounds like a bug. Do you have a reduced test case somewhere so I can take a look?

**Comment by drewB** [comment]
Here you go http://jsfiddle.net/sz2dcf93/6/

**Comment by bramstein** [comment]
Thanks! For my own record, and even more reduced test case: http://jsfiddle.net/xc15jedj/2/

**Comment by NekR** [comment]
Seems like this is because default font in IE has same with for that symbol as you have in your font. This is why fails.

**Comment by ilanle** [comment]
So what is happening with this bug?
